meson.build

```diff
@@ -55,7 +57,9 @@ if get_option('utils')
     subdir('utils')
 endif
 
-subdir('py')
+if get_option('pykms').enabled()
+    subdir('py')
+endif
 
```

Comment:
Sorry, was a quick fix for the buildroot case ;-), dropped this one for pybind11 required dependency only if pykms is enabled...

Reply:
It still doesn't work right. I don't have pybind11 installed on my system, and if I do:

`meson build -Dsystem-pybind11=true -Dpykms=auto`

Meson passes fine, but when building it'll fail as pybind11 is not found. This is because the "auto" setting doesn't specify if the dependency is actually needed or not.

This probably needs a bit more work to make it work right. Maybe the whole pybind11_dep stuff should be moved from the top level meson.build to the pykms meson.build. I think I added it to the top level meson.build just because it's referring to the ext/pybind11/ directory, and it felt kind of more natural to declare pybind11 "module" at the top level, instead of pointing to it from the py/pykms/ subdirectory.

But that's not a real issue, and as moving the pybind11_dep to pykms meson.build solves this one being discussed, I guess that's the way to go.

Btw, I guess you didn't notice, but some time back I sent "[PATCH v2 1/1] package/kmsxx: bump to latest" to buildroot list. These patches of yours will probably make the bump to latest a bit cleaner. So lets get kms++ upstream cleaned up properly so we'll get a nice, clean package to buildroot =).

Reply:
Your are right (only tested with -Dpykms=disabled), updated patch to build pykms only in case pybind11 dependency is found...

Sorry missed/forgot about your patch, but will take a closer look at it later (renaming an option needs Config.in.legacy handling, adding pykms should be done in an extra/follow-up patch)...